fund_store/db/queries.py

```diff
@@ -277,7 +277,7 @@ def insert_fund_data(fund_config, commit: bool = True):
         "owner_organisation_name": fund_config["owner_organisation_name"],
         "owner_organisation_shortname": fund_config["owner_organisation_shortname"],
         "owner_organisation_logo_uri": fund_config["owner_organisation_logo_uri"],
-        "ggis_scheme_reference_number": fund_config["ggis_scheme_reference_number"],
+        "ggis_scheme_reference_number": fund_config["ggis_scheme_reference_number"] if 'ggis_scheme_reference_number' in fund_config else None,
```

Comment:
```suggestion
        "ggis_scheme_reference_number": fund_config.get("ggis_scheme_reference_number"),
```

maybe this alternative is a bit more concise/pythonic, but both should be fine 👍

Reply:
@albertkol @samuelhwilliams merge which is good :D